src/pages/home.css

```diff
@@ -0,0 +1,212 @@
+#page {
+	margin-left: 40px;
+	margin-right: 40px;
+	display: block;
+	margin-top: 49px;
+}
+
+/* #region Page Header */
+
+#page-header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+}
+
+#page-header-main {
+	display: flex;
+	align-items: center;
+}
+
+#page-header-main-generic-profile-icon {
+	width: 40px;
+	height: 40px;
+	background-color: orange;
+	border-radius: 50px;
+}
+
+#page-header-main-text {
+	width: 100%;
+	padding-left: 10px;
+
+	font-family: 'Inter';
+	font-style: normal;
+	font-weight: 700;
+	font-size: 39px;
+
+	color: #4b4b4b;
+}
+
+#page-header-button {
```

Comment:
All of the #page-header-button-XXX stuff should be able to be merged into one CSS rule and just applied to the div all of the content is in. I suggest using maybe a template grid. Also, you only need to apply the color to the text since the SVG is already the correct color since it comes from Figma. 

This should be done for any similarly styled-components as well.

Reply:
Thanks for the suggestion. The code looks a lot cleaner!